test/TestUtil.ts

```
@@ -44,59 +46,48 @@
+          return resolve(rows.rows);
+        }
         return resolve(rows);
       });
     });
   }
 
   static getConnection() {
     if (!this.connection) {
-      const config: any = this.getMySqlConfig();
+      const config = this.getDatabaseConfig();
       if (process.env.CI) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)